### PR TITLE
Puts the addiction amulets on your wrists instead of your neck

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Title + grammar fix on the master rod

## Why It's Good For The Game

There's way more jobs that have their necks full with other items, thus unable to proc having the amulet, this grants those jobs their amulets, at the cost of the few, wrist full jobs.

Grammar Check.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by sanshoom** [comment]
holy shit thank you

File: code/game/objects/items/rogueweapons/melee/special.dm [hunk 1]
```diff
@@ -78,13 +78,13 @@
 				return
 
 			if(istype(user.used_intent, /datum/intent/lord_electrocute))
-				HU.visible_message("<span class='warning'>[HU] electrocutes [H] with the [src].</span>")
+				HU.visible_message("<span class='warning'>[HU] electrocutes [H] with [src].</span>")
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueweapons/melee/special.dm:81]:
```suggestion
				HU.visible_message("<span class='warning'>[HU] electrocutes [H] with \the [src].</span>")
```

File: code/game/objects/items/rogueweapons/melee/special.dm [hunk 2]
```diff
@@ -78,13 +78,13 @@
...
 				H.electrocute_act(5, src)
 				to_chat(H, "<span class='danger'>I'm electrocuted by the scepter!</span>")
 				return
 
 			if(istype(user.used_intent, /datum/intent/lord_silence))
-				HU.visible_message("<span class='warning'>[HU] silences [H] with the [src].</span>")
+				HU.visible_message("<span class='warning'>[HU] silences [H] with [src].</span>")
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueweapons/melee/special.dm:87]:
```suggestion
				HU.visible_message("<span class='warning'>[HU] silences [H] with \the [src].</span>")
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Always rember the byond manual has handy stuff to prevent double "the" 's

**Review by ThePainkiller — APPROVED** [review]

**Pull request merged**